mysql-test/suite/rocksdb/t/disabled.def

```diff
@@ -1,6 +1,6 @@
 compression_zstd : MYR-28 - Percona Server requires compilation with zstd
-cons_snapshot_read_committed : MYR-124 - Percona Server with MyRocks has different behavior of STWCS
-cons_snapshot_repeatable_read : MYR-124 - Percona Server with MyRocks has different behavior of STWCS
+#cons_snapshot_read_committed : MYR-124 - Percona Server with MyRocks has different behavior of STWCS
```

Comment:
Good catch, those were supposed to be removed. The comment was temporary during testing so I could remember exactly which tests needed work. I'll fix and re-push before merging.